### DISCUSSION TITLE
Hỏi về chế độ (Mode) của đèn và quạt.

### DISCUSSION THREAD

**Discussion by nguyencharlie** [khoavukhoa3004/DADN_TTNT#7] [Q&A] [answered]
Ask (Device & Sensor): Đối với các thiết bị như quạt, đèn có nên thêm một Button là CHẾ ĐỘ (Mode) không? Và nếu có thì chức năng cho từng thiết bị đó sẽ như thế nào?

**Comment by khoavukhoa3004** [comment, marked as answer]
Tui nghĩ cái mode đó sẽ ở trong trang quạt, đèn riêng. Với mode của quạt tương ứng là tốc độ xoay (mạnh, vừa, hay nhẹ) còn với đèn thì theo ông Đăng nói, sẽ là các loại màu (giới hạn một số kiểu màu).

**Reply by nguyencharlie** [reply]
Vậy thì có thể tui sẽ chỉnh sửa lại UI (không làm cuộn tròn nữa), mà có thể sẽ làm kiểu khác.
